iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -454,6 +454,8 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
             if (Logging.IsEnabled)
                 Logging.Enter(this, cancellationToken, nameof(CloseAsync));
 
+            _isOpen = false;
```

Comment:
should this flag be set after it has been disposed?

Reply:
It already sets this flag to false, yeah.